src/ctapipe/image/muon/tests/test_ring_fitter.py

```diff
@@ -11,9 +11,19 @@ def test_MuonRingFitter_has_methods():
     assert len(MuonRingFitter.fit_method.values) >= 2
 
 
-@pytest.mark.parametrize("method", MuonRingFitter.fit_method.values)
-def test_MuonRingFitter(method, prod5_mst_flashcam):
+@pytest.mark.parametrize(
+    "method, ringAsymmetry_slope_x, ringAsymmetry_slope_y",
+    [
+        (MuonRingFitter.fit_method.values[0], 1.3, 1.3),
```

Comment:
How did you select the asymmetry values and why they are different for different fit methods?